Exiled.API/Features/Player.cs

```diff
@@ -1090,6 +1088,20 @@ public static Player Get(string args)
         /// <param name="camera">The <see cref="Camera079"/> object to switch to.</param>
         public void SetCamera(Camera079 camera) => SetCamera(camera.cameraId);
 
+        /// <summary>
+        /// Sets the player's level.
+        /// Only applies if the player is SCP-079.
+        /// </summary>
+        /// <param name="newLevel">The level to be set.</param>
+        /// <param name="displayNewLevel">Indicates whether the message should be displayed.</param>
+        public void SetLevel(byte newLevel, bool displayMessage)
```

Comment:
This method does the same thing as the `Level` property, except for `displayMessage`, it's almost like a duplicate, I don't think that's needed.